live-examples/html-examples/content-sectioning/css/header.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+    font-family: "Dancing Script";
+    src: url('/media/examples/DancingScript-Regular.ttf');
+}
+
+header.page-header {
+    background: no-repeat left/cover url(/media/examples/puppy-header-logo.jpg);
+    height: 120px;
+    width: 100%;
+  	min-width: 380px;
+    color: #fff;
+    text-shadow: #000 0 0 0.2em;
+  	position: relative;
+}
+
+header.page-header > h1 {
+    font: bold 3em "Dancing Script", cursive, fantasy;
+    position: absolute;
+  	width: 100%;
```

Comment:
Formatting nit: this looks like 2 spaces and a tab, not 4 spaces.

Reply:
Weird. I'll fix these. Not sure how it happened.